zboss: libraries update (#1229)
nrfconnect/sdk-nrfxlib: zboss/CHANGELOG.rst

All notable changes to this project in the |NCS| are documented in this file.

nRF Connect SDK v2.6.0
**********************

Updated
=======

* The ZBOSS stack to version ``3.11.3.0+5.1.4``.
  For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v2.6.0 release.

Removed
=======

* Precompiled development ZBOSS libraries.

Bug fixes
=========

* Fixed security vulnerabilities.
* Fixed an issue with group command with group ID 0xFFF7.
* Fixed an issue with incorrect access type in Zigbee Discover Attributes Extended Response.

> * Fixed an issue with incorrect handling two "move to color" commands received in a row.

Reviewer: ```suggestion
* Fixed an issue with incorrect handling of two "move to color" commands received in a row.
```

> * Fixed an issue with incorrect validation of Manufacturer specific attribute bit.

Reviewer: ```suggestion
* Fixed an issue with incorrect validation of Manufacturer-specific attribute bit.
```

* Fixed an issue with incorrect response of ZC to beacon when joining is open.

> * Fixed an issue that prevented to define individual reporting contexts when multiple endpoints are implemented.

Reviewer: ```suggestion
* Fixed an issue that prevented defining individual reporting contexts when multiple endpoints are implemented.
```
